ios/FluentUI/People Picker/ContactView.swift

```diff
@@ -0,0 +1,189 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+open class ContactView: UIView {
+    private struct Constants {
+        static let contactWidth: CGFloat = 70.0
+        static let avatarViewWidth: CGFloat = 70.0
+        static let avatarViewHeight: CGFloat = 70.0
+        static let titleLabelMinimumHeight: CGFloat = 12.0
+        static let spacingBetweenAvatarAndFirstLabel: CGFloat = 13.0
+    }
+
+    @objc public var avatarImage: UIImage? {
+        didSet {
+            setupAvatarView(with: titleLabel?.text, and: subtitleLabel?.text, or: nil)
```

Comment:
>nil [](start = 82, length = 3)

what if the view uses an identifier and a client changes the image?

Reply:
Addressed